Checkmate2019/Base/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+from .models import Member, Team
+
+
+admin.site.register(Member)
+admin.site.register(Team)
```

Comment:
@ashryaagr: Can you please ensure that all files end with a newline?

Reply:
Take a look at [PEP8 Style Guide](https://www.python.org/dev/peps/pep-0008/) @ashryaagr